src/app/services/mail/mail.utils.ts

```diff
@@ -269,27 +271,21 @@ export const generateSmsVerificationWarningHtmlForCollab = (
 }
 
 export const generatePaymentConfirmationHtml = ({
-  formTitle,
-  submissionId,
-  appName,
-  invoiceUrl,
+  htmlData,
 }: {
-  formTitle: string
-  submissionId: string
-  appName: string
-  invoiceUrl: string
-}): string => {
-  return dedent`
-    <p>Hello there,</p>
-    <p>
-      Your payment on ${appName} form: ${formTitle} has been received successfully.
-      Your response ID is ${submissionId} and your payment invoice can be found 
-      <a href="${invoiceUrl}">here</a>.
-    </p>
-    <p>Regards,
-    <br />
-    ${appName} team</p>   
-  `
+  htmlData: PaymentConfirmationData
+}): ResultAsync<string, MailGenerationError> => {
+  const pathToTemplate = `${process.cwd()}/src/app/views/templates/payment-confirmation.view.html`
+  logger.info({
+    message: 'generatePaymentConfirmationHtml',
+    meta: {
+      action: 'generatePaymentConfirmationHtml',
+      pathToTemplate,
+      __dirname,
+      cwd: process.cwd(),
```

Comment:
/nit its probably not necessary to log those 2 fields, or at least no need to log `__dirname`, since it's not being used in the function.

I can see many log blocks above do log them, but while I cannot remember why we do it 😑, I suspect it was to verify values to move away from `__dirname`. 

I just checked a recent log which give this:
```
action: generateAutoreplyHtml
__dirname:  /opt/formsg/src/app/services/mail
cwd: /opt/formsg
```

So it looks like using `process.cwd()` is indeed safe to use, and we don't have to log the properties any more. And we can probably schedule to upgrade the other functions to consistently use `process.cwd()` in another ticket.

Reply:
ok i will remove them then, was wondering why also, but decided to just follow.